Header header

bool remote				# timeout flag for remote topic
bool body_state			# timeout flag for body state topic (body_state_track, body_state_cam, or body_state_sim)
bool body_setp_pose		# timeout flag for body pose setpoint topic
bool body_setp_wrench	# timeout flag for body wrench setpoint topic
bool motor_setp			# timeout flag for motor setpoints topic
bool motor_state		# timeout flag for motor states topic
bool servo_setp			# timeout flag for servo setpoints topic
bool servo_state		# timeout flag for servo states topic

bool armed				# armed state flag